command_line_assistant/utils/cli.py

```diff
@@ -187,6 +187,9 @@ def create_subparser(parser: SubParsersAction, name: str, help: str) -> Argument
         name,
         help=help,
         add_help=False,
+        # If allow_abbrev is set to True, then we can use --from instead of
+        # --from-chat, for example.
```

Comment:
It took me longer than I expected to figure out how this fix addressed the issue. I see now, but I would suggest we alter the comment to be more generic and descriptive.

```suggestion
        # Disable abbreviated argument matching. This prevents partial strings from matching to valid options (for example, --from matching to --from-chat).
```